bitcoin/secp256k1_main.py

```diff
@@ -311,16 +302,18 @@ def ecdsa_raw_sign(msg,
         ndata = ffi.new("char [32]", usenonce)
         usenonce = (nf, ndata)
     if formsg:
-        sig = newpriv.ecdsa_sign_recoverable(msg, raw=rawmsg)
-        s, rid = newpriv.ecdsa_recoverable_serialize(sig)
+        if rawmsg:
+            sig = newpriv.sign_recoverable(msg, hasher=None)
+        else:
+            sig = newpriv.sign_recoverable(msg)
+        s, rid = sig[:64], bytes_to_int(sig[64:])
```

Comment:
On reflection don't worry. The issue is: I added the message signing feature on user request about a month ago, but due to pressure of time haven't added it to the test suite (it's hardly critical anyway). Just wanted to sanity check that the algo still works correctly here (don't forget I am not coming from the position of knowing every line of the coincurve code). I or someone else can get round to checking it later.